clients/ctl/admin-ui/src/features/config-wizard/ReviewSystemForm.tsx

```diff
@@ -93,9 +100,86 @@ const ReviewSystemForm = ({ system, onCancel, onSuccess }: Props) => {
             </ReviewItem>
             <ReviewItem label="System tags">
               {initialValues.tags.map((tag) => (
-                <TaxonomyEntityTag key={tag} name={tag} />
+                <TaxonomyEntityTag key={tag} name={tag} mr={1} />
               ))}
             </ReviewItem>
+            <ReviewItem label="System dependencies">
+              {initialValues.system_dependencies.map((dep) => (
+                <TaxonomyEntityTag key={dep} name={dep} mr={1} />
+              ))}
+            </ReviewItem>
+            {!abridged ? (
```

Comment:
There's a lot of nested ternaries here and it's a little bit hard to read. Is there a way we can break this out to make it a little easier on the eyes? 👀

Reply:
I pulled the extended parts into its own component the way I did for DescribeSystems and PrivacyDeclarations. I avoided it at first since it's a little weird without https://github.com/ethyca/fides/issues/1084, but I agree it makes things more readable, so I went ahead and just did it now 👍